locale/fi/toggle.json

```diff
@@ -82,6 +82,6 @@
   "default.description": "",
   "toggle.label.button": "Esimerkkitoiminto, painike-variaatio",
   "toggle.label.input": "Esimerkkitoiminto, kenttä-variaatio",
-  "toggle.state.on": "{{name}} päällä.",
-  "toggle.state.off": "{{name}} pois päältä."
+  "toggle.state.on": " päällä.",
+  "toggle.state.off": " pois päältä."
```

Comment:
Why there is empty spaces?

Reply:
If you mean on the translation value, it might be because without it the space would have been given in the place where this value is used. And might be that it was thought to be cleaner to be like that.

I suggest no change regarding this PR as it shouldn't have been changed now.